overlay/libexec/k3os/mode-disk

```diff
@@ -111,6 +111,10 @@ takeover()
 
     touch k3os/system/factory-reset
 
+    if [ -L sbin ]; then
+      rm -rf sbin
```

Comment:
@brandond why the `-r` if we know it's a symlink?

Reply:
force of habit...